colossalai/shardformer/policies/gpt2.py

```diff
@@ -194,3 +242,222 @@ class GPT2ForSequenceClassificationPolicy(GPT2Policy):
 
     def __init__(self) -> None:
         super().__init__()
+
+
+class GPT2PipelineForwards:
```

Comment:
so, use class and '' or class and import here ?

Reply:
I will use class and '' for typehint.